src/transformer/utils/get_user.php

```diff
@@ -24,8 +24,9 @@ function get_user(array $config, \stdClass $user) {
 
     if (array_key_exists('send_mbox', $config) && $config['send_mbox'] == true && $hasvalidemail) {
         return [
-            'name' => $fullname,
-            'mbox' => 'mailto:' . $user->email,
+           // 'name' => $fullname,
+			'mbox_sha1sum' => sha1('mailto:' . $user->email),
```

Comment:
It would be better to make a plugin configuration option for enabling/disabling the hashing of emails, in order to improve upstream-ability.

Reply:
Agreed, but can we please proceed with testing as-is while we look into making this a configuration option?

Reply:
I've created #2 to track this (I also noticed that `name` was customised out, so included that there as well).